_ml-commons-plugin/conversational-search.md

```diff
@@ -287,18 +292,21 @@ Use the following steps to set up an HTTP connector using the OpenAI GPT 3.5 mod
 	"connector_id": "f5-iSYoBu0R6vVqGI3PA"
   }
   ``` 
+  {% include copy-curl.html %}
 
 4. With the model registered, use the `task_id` returned in the registration response to get the `model_id`. You'll use the `model_id` to deploy the model to OpenSearch:
 
-  ```json
+  ```curl
```

Comment:
Throughout the page, please make sure that:
1. The requests are formatted correctly (some are squished to the left, others have really big tabs) 
2. Copy buttons are added to all requests
3. You use "json" for code blocks. "curl" is not a supported syntax highlighter in Jekyll.